app/src/main/java/com/github/polypoly/app/map/LocalizationRepository.kt

```diff
@@ -0,0 +1,94 @@
+package com.github.polypoly.app.map
+
+import com.github.polypoly.app.game.Localization
+import com.github.polypoly.app.game.LocalizationLevel
+import com.github.polypoly.app.game.Zone
+import org.osmdroid.util.GeoPoint
+
+/**
+ * Repository for providing the zones and localizations on the map.
+ */
+object LocalizationRepository {
+
+    /**
+     * List of localizations in the EPFL campus area.
+     */
+    private val localizations = listOf(
+        Localization(
+            name = "BC",
+            basePrice = 200,
+            baseTaxPrice = 20,
```

Comment:
The tax price can be computed with the basePrice and the level for example, It makes sure that all taxes are fair

Reply:
Ditto